lib/pharos/host/debian/scripts/repos/pharos_stretch.sh

```diff
@@ -5,6 +5,6 @@ set -eu
 if [ ! -e /etc/apt/sources.list.d/pharos-kubernetes.list ]; then
     curl -fsSL https://bintray-pk.pharos.sh/?username=bintray | apt-key add -
     cat <<EOF >/etc/apt/sources.list.d/pharos-kubernetes.list
-deb https://dl.bintray.com/kontena/pharos-debian stretch main
+deb http://dl.bintray.com/kontena/pharos-debian stretch main
```

Comment:
Why plain http?

Reply:
It allows packages to be cached at proxy (makes tests faster and less error prone to network gitches). I'll revert these and open separate issue/pr later.